### PR TITLE
feat(integrations): remove tech preview banner

### PR DESCRIPTION
**Pull request by pes-spyro-soft-com**
## Issue

https://gravitee.atlassian.net/browse/APIM-6778

## Description

Remove tech preview banners from integrations

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-hgfbwmceoh.chromatic.com)

### CODE REVIEW

**Comment by phiz71** [comment]
Are you sure you want to remove tech-preview for ALL features n the console ?

File: gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html [hunk 1]
```diff
@@ -402,9 +402,6 @@ <h3 class="portal__form__card__title">Documentation</h3>
         <h2>New Developer Portal</h2>
         <mat-card class="portal__form__card">
           <mat-card-content formGroupName="portalNext">
-            <gio-banner-info>
-              {{ integrationsService.bannerMessages.techPreview }}
-            </gio-banner-info>
```
Comment (phiz71) [inline gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html]: Please, do no remove tech preview for portal stuff 🙏🏻

**Review by phiz71 — CHANGES_REQUESTED** [review]

File: gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html (hunk 1 above)
Comment (pes-spyro-soft-com) [reply gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html]: its a good point, however this is "tightly coupeled" with integrations. I am removing tech preview for integrations, so this should be removed form integrations service.
 If we need "tech preview" for any different functionality(-ties) we need to move it out from integration service to more generic place.

Comment (pes-spyro-soft-com) [reply gravitee-apim-console-webui/src/management/settings/portal-settings/portal-settings.component.html]: I scoped my changes only to removing tech preview banner form integrations pages.

**Review by michel-barret — APPROVED** [review]

**Review by phiz71 — APPROVED** [review]

**Pull request merged**